modules/ROOT/pages/microsoft-sharepoint-2010-connector.adoc

```diff
@@ -423,7 +482,7 @@ For example, calling *resolve* method on the "LookupField" of the above item ret
 }
 ----
 
-* Checked: The full object graph retrieves. In case there is a cycle, the summary reference object is shown:
+Checked: The full object graph retrieves. In case there is a cycle, the summary reference object is shown:
```

Comment:
"the full object graph retrieves"?
```suggestion
Checked: The full object that the graph retrieves
In case there is a cycle, the summary reference object is shown:
```